packages/material-ui/src/Tooltip/Tooltip.test.js

```diff
@@ -84,88 +77,74 @@ describe('<Tooltip />', () => {
 
   describe('prop: placement', () => {
     it('should have top placement', () => {
-      const wrapper = shallow(<Tooltip {...defaultProps} placement="top" />);
+      const wrapper = mount(<Tooltip {...defaultProps} placement="top" />);
       assert.strictEqual(wrapper.find(Popper).props().placement, 'top');
     });
   });
 
   it('should respond to external events', () => {
-    const wrapper = shallow(<Tooltip {...defaultProps} />);
-    wrapper.instance().childrenRef = document.createElement('div');
-    const children = wrapper.childAt(0).childAt(0);
-    assert.strictEqual(wrapper.instance().state.open, false);
-    children.simulate('mouseOver', { type: 'mouseover' });
-    assert.strictEqual(wrapper.instance().state.open, true);
-    children.simulate('mouseLeave', { type: 'mouseleave' });
-    assert.strictEqual(wrapper.instance().state.open, false);
+    const wrapper = mount(<Tooltip {...defaultProps} />);
+    const children = wrapper.find('#testChild');
+    assert.strictEqual(wrapper.find(Popper).props().open, false);
```

Comment:
Could we check for the existence/display state of `[role="tooltip"]`? I suppose that's what we're actually interested in: Is the tooltip shown? and not "Does the component responsible for the tooltip have its open prop set to true?"